src/main/scripts/postToSlackCustom.groovy

```diff
@@ -21,35 +21,50 @@ final def props = airTool.getStepProperties()
 
 // properties
 final def webhook = props['webhook'];
-final def slackChannel = props['channel'];
+final def slackChannels = props['channels'];
```

Comment:
You'll want to migrate the channel property to the channels property so no data is lost during upgrade.

You need to add a quick migration tag regardless in the upgrade.xml, but it should look like this...

`<migrate to-version ="4">`
` <migrate-command name="Post Custom Notification to Slack">`
`<migrate-properties>`
`<migrate-property name="channels" old="channel" default="" />`
`</migrate-properties>`
`</migrate-command>`
`</migrate>`